drivers/windows/R/paths.R

```diff
@@ -68,7 +68,7 @@ print.windows_path <- function(x, ...) {
 
 remote_path <- function(x, shares) {
```

Comment:
This function wasn't being used anywhere outside of its own unit tests, so I've re-used the name and changed its semantics a bit to fit what I wanted.